pkg/k8s_tap_service.go

```diff
@@ -23,6 +25,11 @@ import (
 	"github.com/turbonomic/kubeturbo/pkg/discovery/monitoring/master"
 )
 
+const (
+	usernameFilePath = "/etc/opsmgr-credentials/username"
+	passwordFilePath = "/etc/opsmgr-credentials/password"
```

Comment:
please do not call it opsmgr- just call it turbo credentials

Reply:
Updated!